.github/workflows/test.yml

```diff
@@ -7,12 +7,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ '3.6', '3.7', '3.8' ]
+        python-version: [ '3.7', '3.8' ]
```

Comment:
we should add also 3.9 and 3.10. In another PR

Reply:
For the moment that's not urgent. Thing is that py3.6 is blocking service tags